deployment/ccip/changeset/cs_deploy_token_pools.go

```diff
@@ -151,7 +159,7 @@ func DeployTokenPoolContractsChangeset(env deployment.Environment, c DeployToken
 		chain := env.Chains[chainSelector]
 		chainState := state.Chains[chainSelector]
 
-		_, err := DeployTokenPool(env.Logger, chain, chainState, newAddresses, poolConfig)
+		_, err := DeployTokenPool(env.Logger, chain, chainState, newAddresses, poolConfig, c.UseTestRouter)
```

Comment:
nit : probably easier to take router address as input param?

should DeployTokenPool be unexported?

Reply:
It should not: https://github.com/smartcontractkit/chainlink/pull/16178/commits/07f41ae148929c69cf98e26d5f2e23de9618494a